docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1695,6 +1695,33 @@ Office.RecurrenceTimeZone:interface:
         "seriesTime": {seriesTimeObject},
         "recurrenceTimeZone": {"name": "Pacific Standard Time", "offset": -480}
     }
+Office.SensitivityLabelDetails:interface:
```

Comment:
What's the reason for adding the snippets here instead of in Script Lab? Not a critique, just curious.

Reply:
I was initially going to use the existing Script Lab snippet, but decided to create a separate one in the reference repo based on past user feedback. The current snippet logs the `asyncResult.value` object without printing each individual sensitivity label property. The aim was to only highlight what the method does without overwhelming the user with individual property information. Past feedback indicated that if we used that same sample in something like the Office.SensitivityLabelDetails interface, it wasn't immediately clear what message the sample was trying to convey without logging each indidividual property to the console. We could look into integrating this into the Script Lab snippet in the future (I don't think we received any particular feedback on that. We just didn't want to overwhelm the user's console.).

Reply:
Makes sense! Thanks